sprocs/index.js

```diff
@@ -33,8 +33,6 @@ module.exports = {
             'assessments_format.sql',
             'assessments_format_for_question.sql',
             'tags_for_question.sql',
-            'assessment_instances_points_homework.sql',
```

Comment:
I am a little confused. I think you are eliminating the use of `assessment_instances_points_homework.sql` and of `assessment_instances_points_exam.sql` in favor of using only `assessment_instances_points.sql`. If so, why do the two old `.sql` files still exist? Why didn't you delete them?

Reply:
Good catch. This was an oversight.